### DISCUSSION TITLE
Include debug logging script in release zips

### DISCUSSION THREAD

**Discussion by ted-miller** [Yaskawa-Global/motoros2#311] [Ideas]
@gavanderhoorn Regarding [this comment](https://github.com/Yaskawa-Global/motoros2/pull/309#discussion_r1757479677).

> we actually don't include the logger script in release .zips

Why don't we? I think it could be beneficial to include it with a short README explaining how to use it.

**Comment by gavanderhoorn** [comment]
This is likely an oversight.

The idea probably was that the debug logger would mostly be used by developers, not regular users.

Since then we've actually started using it as a generic support tool, so perhaps we should revisit that.
